Application.py

```diff
@@ -0,0 +1,73 @@
+import Add_User as A
+import Delete_User as B
+import Check_Balance as C
+import Check_Address as D
+import Edit_Details as E
+import Update_Balance as F
+import Same_Name as G
+import More_Money as H
+import Show_Money as I
+
+class Account:
+    def __init__(self, name, address,balance,ID):
+        self.name = name
+        self.address = address
+        self.balance = balance
+        self.ID = ID
+
+
+def main():
+    Quit = False
+    print("WELCOME TO OUR BANKING APP \n \n")
+    while Quit == False:
+        Quit = menu()
+    print("\n")
+    print("We are sorry to see you go. Have a nice day :)")
+    
+
+
+def menu():
+    print("How can we help you today? \n")
```

Comment:
Horrible for my eyes:
menu_message= "Bla bla /n Bla bla /n Bla bla /n "
print(menu_message)